pysrc/fetch/dune.py

```diff
@@ -67,41 +64,20 @@ async def fetch(self, query: Query) -> list[DuneRecord]:
             self.log.error(f"Got {err} - Exiting")
             sys.exit()
 
-    async def app_hash_block_range(self, fname: str, column: str) -> BlockRange:
+    async def latest_app_hash_block(self) -> int:
         """
         Block Range is used to app hash fetcher where to find the new records.
         block_from: read from file `fname` as a loaded singleton.
             - uses genesis block is no file exists (should only ever happen once)
             - raises RuntimeError if column specified does not exist.
         block_to: fetched from Dune as the last indexed block for "GPv2Settlement_call_settle"
         """
-        # Genesis block:
-        block_from = 12153262
-        try:
-            block_from = int(self.file_manager.load_singleton(fname, "csv")[column])
-        except FileNotFoundError:
-            self.log.warning(
-                f"block range file {fname}.csv not found, using genesis block {block_from}"
-            )
-        except KeyError as err:
-            message = (
-                f"block range file {fname}.csv does not contain column header {column}, "
-                f"exiting to avoid duplication"
-            )
-            self.log.error(message)
-            raise RuntimeError(message) from err
-
-        return BlockRange(
-            # TODO - could be replaced by Dune Query on the app_data table (once available).
-            #  https://github.com/cowprotocol/dune-bridge/issues/42
-            block_from,
-            block_to=int(
-                # KeyError here means the query has been modified and column no longer exists
-                # IndexError means no results were returned from query (which is unlikely).
-                (await self.fetch(QUERIES["LATEST_APP_HASH_BLOCK"].query))[0][
-                    "latest_block"
-                ]
-            ),
```

Comment:
This filemanager stuff didn't belong in the Dune Fetcher. Now this method only makes a dune query. The `from_block` logic has been moved to `sync/app_data.py`